src/components/Nav/Nav.js

```diff
@@ -5,9 +5,9 @@ import './Nav.scss'
 const Nav = () => {
   return (
     <nav className='Nav'>
-      <NavLink to='/' name='recent'>Recent</NavLink>
-      <NavLink to='/favorites' name='favorites'>Favorite Movies</NavLink>
-      <NavLink to='/popular' name='popular'>Popular Movies</NavLink>
+      <NavLink exact to='/' activeStyle={{ color: 'red' }} name='recent'>Recent</NavLink>
+      <NavLink exact to='/favorites' activeStyle={{ color: 'red' }} name='favorites'>Favorite Movies</NavLink>
+      <NavLink exact to='/popular'  activeStyle={{ color: 'red' }} name='popular'>Popular Movies</NavLink>
```

Comment:
Instead of red could we use a color from our original palette? After Robbie's comment about red === bad for user experience I think one of our softer colors could be a good choice! 🌈

Reply:
I think we can roll this up on a commit that will also solve: #41

Reply:
issue #42